test/tests/meshgenerators/file_mesh_generator/2d_discontinuous_iga.i

```diff
@@ -0,0 +1,53 @@
+[Mesh]
+  [cyl2d_iga]
+    type = FileMeshGenerator
+    file = PressurizedCyl_Patch6_4Elem.e
+    discontinuous_spline_extraction = true
+  []
+  allow_renumbering = false
+  parallel_type = replicated
```

Comment:
should all these limitations be baked in the code somewhere? 
I see max_parallel of 1 as well in the test

Reply:
The allow_renumbering=false is a limitation of XMLDiff, not of our code here.  But right now we're using exodiff thanks to *other* XMLDiff issues, so I'll reenable renumbering.

I've fixed the parallel and distributed-parallel issues now.